### PR TITLE
Add markdown spellchecking

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cirrus
+ci
+cirruslabs
+kubernetes
+google
+engine
```
Comment (fkorotkov) [inline .spelling]: `engine` should be fine?

Comment (RDIL) [reply .spelling]: fixed

**Comment by fkorotkov** [comment]
I noticed that logs are not very instructive about which word caused the error. Is there a way to make spelling reporting nicer?

**Comment by RDIL** [comment]
Not really.  I’m working on making this a Pester test over at powershell, maybe once I’m finished and have legal consent I can add it here.

**Comment by RDIL** [comment]
(But generally the word that caused the failure is in the middle of the text displayed. )

**Comment by RDIL** [comment]
@fkorotkov done!

**Comment by fkorotkov** [comment]
Let me think about it for a while. `.spelling` seems a little bit too big and complex. I wonder if it will become a problem to maintain it.

**Comment by fkorotkov** [comment]
@RDIL let's sort `.spelling` alphabetically and merge it.

**Pull request merged**